src/components/CredentialForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+
+const CredentialForm = ({ setErrors }) => {
```

Comment:
`setErrors` prop here could be removed since its not used and probably was passed as props previously.

Reply:
I've reviewed this PR, looks good. After addressing the above comment, feel free to merge 🥳